foundry/pixi/containers/canvasLayers/placeablesLayers/drawingsLayer.d.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * The DrawingsLayer subclass of PlaceablesLayer
+ *
+ * This layer implements a container for drawings which are rendered immediately above the TilesLayer
+ * and immediately below the GridLayer.
+ */
+declare class DrawingsLayer extends PlaceablesLayer<Drawing> {
+  /**
+   * @override
+   * @defaultValue
+   * ```
+   * mergeObject(super.layerOptions, {
+   *   canDragCreate: true,
+   *   canDelete: true,
+   *   controllableObjects: true,
+   *   rotatableObjects: true,
+   *   objectClass: Drawing,
+   *   sheetClass: DrawingConfig,
+   *   zIndex: 20
+   * })
+   * ```
+   */
+  static get layerOptions(): PlaceablesLayer.LayerOptions;
+
+  // Configuration Setting
+  /**
+   * @defaultValue `"defaultDrawingConfig"`
+   */
+  static DEFAULT_CONFIG_SETTING: string;
```

Comment:
I can move it to the bottom, but I think  they had me put a static value near the parameters before. Clarification would be nice, because I'm probably misremembering